controllers/articles-controller.js

```diff
@@ -1,13 +1,24 @@
-const { fetchArticleById } = require('../models/articles-model')
+const { fetchArticleById, fetchArticles } = require("../models/articles-model");
 
 const getArticleById = (request, response, next) => {
-    const articleId = Number(request.params.article_id)
-    fetchArticleById(articleId).then((article) => {
-        response.status(200).send({article})
+  const articleId = Number(request.params.article_id);
+  fetchArticleById(articleId)
+    .then((article) => {
+      response.status(200).send({ article });
     })
     .catch((err) => {
-        next(err)
+      next(err);
+    });
+};
+
+const getArticles = (request, response, next) => {
+  fetchArticles()
+    .then((articles) => {
+      response.status(200).send({articles});
     })
-}
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Good work here, the controller and model function names are good. Sending the response back on an object with a key of `articles` is spot on, well done.